extern/test_bmi_py/bmi_model.py

```diff
@@ -79,8 +79,8 @@ def __init__(self):
                            'OUTPUT_VAR_2':['OUTPUT_VAR_2','mm/s'],
                            'OUTPUT_VAR_3':['OUTPUT_VAR_3','-'],
                            'GRID_VAR_1':['OUTPUT_VAR_1','mm/s'],
-                           'GRID_VAR_2':['GRID_VAR_2','-'],
-                            }
+                           'GRID_VAR_2':['GRID_VAR_2','-']
+                           }
```

Comment:
Look at `_output_var_names` just above - that lists `GRID_VAR_2` and `GRID_VAR_3`. The latter is missing here. Please add it.

Reply:
This seems to be the cause of all remaining unit test failures